draft-ietf-quic-http.md

```diff
@@ -1102,69 +1167,33 @@ allowing time for any in-flight requests (at least one round-trip time), the
 server MAY send another GOAWAY frame with an updated last Stream ID.  This
 ensures that a connection can be cleanly shut down without losing requests.
 
-Once all requests on streams at or below the identified stream number have been
-completed or cancelled, and all promised server push responses associated with
-those requests have been completed or cancelled, the connection can be closed
-using an Immediate Close (see {{QUIC-TRANSPORT}}).  An endpoint that completes a
-graceful shutdown SHOULD use the QUIC APPLICATION_CLOSE frame with the
-HTTP_NO_ERROR code.
+Once all accepted requests have been processed, the server MAY permit the
+connection to become idle, or MAY initiate an explicit closure of the
+connection.  An endpoint that completes a graceful shutdown SHOULD use the
+HTTP_NO_ERROR code when closing the connection.
 
+## Explicit Application Closure
```

Comment:
Explicit or immediate?  Better to use the same terminology as the transport doc, methinks.  Explicit, taken to mean "direct", might be right, but it seems a little off.